docs/project/test-and-docs.md

```diff
@@ -185,6 +185,29 @@ Most test targets require that you build these precursor targets first:
 
 ## Running individual or multiple named tests 
 
+### Unit tests 
+
+We use golang standard [testing](https://golang.org/pkg/testing/)
+package or [gocheck](https://labix.org/gocheck) for our unit tests. 
+
+You can also use the `TESTDIRS` environment variable to run unit tests for
```

Comment:
Oh I think I messed up there.. using "also" here, and nothing in the next paragraph, I'll update that :sweat_smile: